sybil/sybil.py

```diff
@@ -1,10 +1,18 @@
 import inspect
 import sys
 from pathlib import Path
-from typing import Sequence, Callable, Collection, Mapping, Optional, Type, List
+from typing import Sequence, Callable, Collection, Mapping, Optional, Type, Dict, Any, List, TYPE_CHECKING
+from unittest import TestSuite
+from unittest.loader import TestLoader
 
 from .document import Document, PythonDocStringDocument, PythonDocument
 from .typing import Parser
+from .integration.pytest import SybilFile
+
+if TYPE_CHECKING:
+  # pytest is a tests-only requirement, so we cannot have it as an import outside
+  # of the type checking block.
+  from _pytest.nodes import Collector
```

Comment:
See https://github.com/simplistix/sybil/issues/82.
If this is not in the `if TYPE_CHECKING` block, the docs build fails.